.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "SpongeAPI"]
 	path = SpongeAPI
-	url = https://github.com/SpongePowered/SpongeAPI.git
+	url = https://github.com/JBYoshi/SpongeAPI.git
```

Comment:
Just leave them out. Its not a big deal and easily fixable by whoever merges this but its easily missed and then we have to hurry and make a commit before everyone and their brother ends up pulling your fork XD.